src/constants.js

```diff
@@ -156,4 +156,5 @@ module.exports = {
         VISIONPRESCRIPTION: 'VisionPrescription',
         CLIENT: 'Client',
     },
+    KAFKA_CONNECTION_TIME_OFFSET: 30000,
```

Comment:
1. Mention the time unit.
2. KAFKA_CONNECTION_HEALTHCHECK_INTERVAL name is more apt.